.travis.yml

```diff
@@ -12,7 +12,9 @@ services:
 script:
   - bundle exec rake rubocop
   - bundle exec rake
+  - COVERBAND_HASH_REDIS_STORE=t bundle exec rake
```

Comment:
love the env toggle on tests, been doing this while rolling out dual strategies at work